### PR TITLE
Add support for Navitia fr-sw region (which covers South-West of France)

### PR DESCRIPTION
**Pull request by kalon33**
This commit should add proper initial support for Navitia fr-sw region. Further testing may be needed to check proper tram/bus... logotypes rendering.

### CODE REVIEW

File: enabler/src/de/schildbach/pte/FrenchSouthWestProvider.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+/**
+ * @author Nicolas Derive
+ */
+public class FrenchSouthWestProvider extends AbstractNavitiaProvider
+{
+	private static String API_REGION = "fr-sw";
+
+	public FrenchSouthWestProvider(final String authorization)
+	{
+		super(NetworkId.ALPC, authorization);
```
Comment (schildbach) [inline enabler/src/de/schildbach/pte/FrenchSouthWestProvider.java]: ALPC? That doesn't match with the new id in NetworkId.java.

**Comment by schildbach** [comment]
Thanks for your contribution! Can you add a FrenchSouthWestProviderLiveTest.java with at least a basic test per query? Have a look at ParisProviderLiveTest.java for an idea.

File: enabler/src/de/schildbach/pte/FrenchSouthWestProvider.java (hunk 1 above)
Comment (kalon33) [inline enabler/src/de/schildbach/pte/FrenchSouthWestProvider.java]: Sorry, I will fix that, I just renamed the class and Network as it doesn't only covers a french administrative region (Aquitaine-Limousin-Poitou-Charentes).

**Comment by kalon33** [comment]
@schildbach  Is that OK now? I wasn't sure of what certain tests were about.

Thanks for your help :)

**Comment by kalon33** [comment]
Do I need to do more editing or is that good?

File: enabler/test/de/schildbach/pte/live/FrenchSouthWestProviderLiveTest.java [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+{
+	public FrenchSouthWestProviderLiveTest()
+	{
+		super(new FrenchSouthWestProvider(Secrets.NAVITIA_AUTHORIZATION));
+	}
+
+	@Test
+	public void nearbyStationsAddress() throws Exception
+	{
+		nearbyStationsAddress(44826434, -0557312);
```
Comment (schildbach) [inline enabler/test/de/schildbach/pte/live/FrenchSouthWestProviderLiveTest.java]: Literals starting with 0 are interpreted as octal. Are you sure this is what you want to do?

**Comment by schildbach** [comment]
Generally looks good. A couple of unit tests fail, but I realize we have this problem with ParisProvider too currently. Maybe these issues should be fixed separately.

File: enabler/test/de/schildbach/pte/live/FrenchSouthWestProviderLiveTest.java (hunk 2 above)
Comment (kalon33) [inline enabler/test/de/schildbach/pte/live/FrenchSouthWestProviderLiveTest.java]: The relevant place coordinates are 44.826434, -0.557312 respectively. Is this malformed?

**Comment by kalon33** [comment]
@schildbach which ones are failing? Why? I might have misunderstood the roles of some tests. I took ParisProvider as a basis to understand them but if ParisProvider is malformed, I may have reproduced the mistakes inside. 

Could you pull it "as is" and I would fix them after? I would need these changes to be integrated to an app I use, Transportr, that use your project and I will travel to the region covered by my PR next week.

Thanks for your help.

File: enabler/test/de/schildbach/pte/live/FrenchSouthWestProviderLiveTest.java (hunk 2 above)
Comment (schildbach) [inline enabler/test/de/schildbach/pte/live/FrenchSouthWestProviderLiveTest.java]: -0557312 is interpreted as octal so it's -188106 decimal. So your coordinate is -0.188106.

And -0580036 is actually an invalid number (overflows) and fails to compile.

**Comment by schildbach** [comment]
Don't worry, I intend to merge this even with failing tests. But it needs to compile…

**Comment by kalon33** [comment]
@schildbach what's needed to properly compile?

**Comment by grote** [comment]
@kalon33 Look at what Andreas was writing here:
> -0557312 is interpreted as octal so it's -188106 decimal. So your coordinate is -0.188106.
> And -0580036 is actually an invalid number (overflows) and fails to compile.

There he explains, why it does not compile.

**Comment by kalon33** [comment]
@grote so what should I write there to correct this?

**Comment by grote** [comment]
@kalon33 I don't have time at the moment to look into this in detail. Just make sure you are passing proper coordinates to the functions. This might also be helpful to understand what Andreas is referring to: https://stackoverflow.com/questions/7218760/why-is-08-not-a-valid-integer-literal-in-java

**Comment by kalon33** [comment]
So -0557312 should be -.557312 and -0580036 should be -.580036?

**Comment by schildbach** [comment]
Just remove the leading zeroes from integers if you intend them to be interpreted as decimal. I can do it for you if you want.

**Comment by kalon33** [comment]
@schildbach I just did it (this was my fault, so I corrected it). So now, -557312 will be considered as -0.557312 longitude coordinate?

**Comment by schildbach** [comment]
Thanks, I squashed and merged your PR.

**Pull request closed without merging**